vcs/vcs.go

```diff
@@ -64,13 +73,9 @@ func GetRepository(dirname string) (string, error) {
 		return "", err
 	}
 	switch vcs {
-	case Git:
+	case Git, Subversion, None:
 		return dir, nil
-	case Subversion:
-		return "", errors.ErrNotImplemented
-	case Mercurial:
-		return "", errors.ErrNotImplemented
-	case Bazaar:
+	case Mercurial, Bazaar:
 		return "", errors.ErrNotImplemented
```

Comment:
[nit] you can change this line to `fallthrough` so that it always hits the default case.

Reply:
thanks—that's a good way of keeping `case Mercurial, Bazaar` explicitly for ourselves while not also having extra code :)